src/Features/LanguageServer/Protocol/Features/Options/WorkspaceConfigurationOptionsStorage.cs

```diff
@@ -40,4 +37,7 @@ public static WorkspaceConfigurationOptions GetWorkspaceConfigurationOptions(thi
 
     public static readonly Option2<bool> EnableOpeningSourceGeneratedFilesInWorkspaceFeatureFlag = new(
         "dotnet_enable_opening_source_generated_files_in_workspace_feature_flag", WorkspaceConfigurationOptions.Default.EnableOpeningSourceGeneratedFiles);
+
+    public static readonly Option2<bool> RunSourceGeneratorsExplicitly = new(
+        "dotnet_run_source_generators_explicitly", WorkspaceConfigurationOptions.Default.RunSourceGeneratorsExplicitly);
```

Comment:
@tmat do i need to do anything to ensure that this works if the user sets this in their editorconfig?